### ISSUE TITLE
Crash when using php-fpm fcgi api without HTTP server

### ISSUE THREAD

**Issue by timfennis** [getsentry/sentry-symfony#494]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
```
sentry/sdk 3.1.0
sentry/sentry 3.2.1
sentry/sentry-symfony 4.1.0
```

### Steps to Reproduce

We have a healthcheck script that monitors the status of php-fpm like so:

```
REQUEST_METHOD=GET REQUEST_URI=/_health SCRIPT_FILENAME=index.php cgi-fcgi -bind -connect 127.0.0.1:9000
```
In this case the call to `$request->getProtocolVersion()` returns `null` which results in an error in `TracingRequestListener::getHttpFlavor`

Error:
```
str_starts_with(): Argument #1 ($haystack) must be of type string, null given
```

### Expected Result

Eventhough this is a weird edge case that probably won't occur often maybe fall back to some empty value?

### Actual Result

I get an error (in sentry though :+1:)

Error:
```
str_starts_with(): Argument #1 ($haystack) must be of type string, null given
```

**Comment by timfennis** [comment]
For now I've fixed this by adding `SERVER_PROTOCOL=HTTP/1.0` to my healthcheck

**Comment by ste93cry** [comment]
We can definitely handle the absence of this key, and the fix would be so simple that a contribution is welcome. Please note however that according to [RFC 3875 (GCI spec)](https://tools.ietf.org/html/rfc3875#section-4.1.16) the key **must** be present, so I would say that you should anyway always pass it if you directly call the script via php-fpm.
